common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -484,6 +485,9 @@ def parse_query_params(strategy, response, *args, **kwargs):
     auth_entry = strategy.request.session.get(AUTH_ENTRY_KEY, AUTH_ENTRY_LOGIN)
     if auth_entry not in _AUTH_ENTRY_CHOICES:
         raise AuthEntryError(strategy.request.backend, 'auth_entry invalid')
```

Comment:
Thanks for moving this set_custom_attribute.  Seeing this error code, it might make sense to:
1. Update the error to `f'auth_entry {auth_entry} invalid'`, or
2. Move the set_custom_attribute above this error, or
3. Both (my recommendation)